### PR TITLE
never mock quibble

### PR DESCRIPTION
**Pull request by connorjclark**
If `quibble` happens to be imported after initial use, it essentially resets the `global.__quibble` state resulting in either total failure (cannot access 'get' of undefined) or a module not being mocked (such as for builtins like fs)

To prevent this, simply return early in `resolve` if the specifier is `quibble`.

### CODE REVIEW

**Comment by searls** [comment]
Humorous, but I can believe it's necessary. Any chance this means it should also be ported to [lib/quibble.js](https://github.com/testdouble/quibble/blob/main/lib/quibble.js)?

**Comment by connorjclark** [comment]
Not sure what you mean, can you rephrase? I think this is only an issue for the loader api / esm, but I don't have as good a mental model of the commonjs route as I do for the loader/ESM.

**Comment by searls** [comment]
Right on, was just curious if this might also be a risk for commonJS quibbling, but what you have looks good

**Comment by searls** [comment]
Landed in v0.6.13
